### ISSUE TITLE
MOBILE Search Results and Pagination - 1.4.10 Reflow - Search results and pagination completely obscure map on mobile.

### ISSUE THREAD

**Issue by imogenroberts** [archesproject/arches#9727]
As a user on either iOS or Android, it is to be expected that all content reflows into a single column, with none of the content obscured by other content or cut off by the edge of the page.

**Linked Work Item:**
#9659 

**Expected Result:**
All content should reflow into a single column on mobile (no two-dimensional scrolling). All content should be visible and accessible; nothing should be obscured by other content or cut off by the edges of the page.

**Actual Result:**
Search results and pagination completely obscure map on mobile. Right hand side of the component is also marginally cut off, though this does not obscure any of the text/links. When entering a valid search term and selecting a result from the drop-down suggestions, the map tabs ('Map Filter', etc.) become visible but are partially obscured.

**Evidence:**

![Image](https://github.com/archesproject/arches/assets/77161085/728f7a12-00b7-4126-ad5e-a49728fbb894)
![Image](https://github.com/archesproject/arches/assets/77161085/e7c05577-6fb2-463c-af7b-7c435900bed4)

**Comment by johnatawnclementawn** [comment]
Resolved in #9741

**Comment by apeters** [comment]
@imogenroberts What branch are you on?  It should be the accessibility branch.  Also make sure you run `yarn build_development` after you pull the latest.  
Here's what I see with the latest.


![Image](https://github.com/archesproject/arches/assets/823773/7a329dcb-a014-482e-8190-56d214e20eef)

**Comment by imogenroberts** [comment]
Ah yes that'll be the problem! I haven't pulled the latest updates. I'll get Paul to help me with that this morning.
